README.md

```diff
@@ -66,12 +66,18 @@ errorMessages={{
   'min.password': 'Custom min password required error message.'
 }}
 ```
+**onValues**
 
-**initialValues**
+exposes the values on change, you must manage the form state by passing in values if using this.
```

Comment:
I might say `passing in state to the ``values`` prop if using this.` to make it more obvious.